### PR TITLE
Add Syncswap on Scroll to dex.trades

### PR DESCRIPTION
**Pull request by Jam516**

### CODE REVIEW

**Comment by 0xRobin** [comment]
@Jam516 before a source can be used in dbt we need to define it in a yaml file.

You'll probably want to replicate this:
https://github.com/duneanalytics/spellbook/blob/d8ccd9634ae2b036bd8a6e2950d5c6c7be8b37e0/sources/_sector/dex/trades/zksync/_sources.yml#L20-L33
for scroll in `sources/_sector/dex/trades/scroll/_sources.yml`

**Comment by Jam516** [comment]
Hi @jeff-dude @Hosuke , this is ready for review

File: models/_sector/dex/trades/scroll/_schema.yml [hunk 1]
```diff
@@ -72,3 +72,20 @@ models:
             - evt_index
       - check_dex_base_trades_seed:
           seed_file: ref('scrollswap_scroll_base_trades_seed')
+
+  - name: syncswap_v1_scroll_base_trades
```
Comment (Hosuke) [inline models/_sector/dex/trades/scroll/_schema.yml:76]: We may need to add `syncswap_v1_scroll_base_trades` into `dex_scroll_base_trades`:
https://github.com/duneanalytics/spellbook/blob/main/models/_sector/dex/trades/scroll/dex_scroll_base_trades.sql

**Review by Hosuke — CHANGES_REQUESTED** [review]

File: models/_sector/dex/trades/scroll/_schema.yml (hunk 1 above)
Comment (Jam516) [reply models/_sector/dex/trades/scroll/_schema.yml:76]: that makes sense, ive added it now

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @Jam516 

[Sample syncswap_v1_scroll_base_trades](https://dune.com/queries/3750989)

**Review by jeff-dude — APPROVED** [review]
looks good to me!

**Pull request merged**